packages/manager/modules/hub/src/components/billing-summary/controller.js

```diff
@@ -0,0 +1,72 @@
+import get from 'lodash/get';
+
+export default class ManagerHubBillingSummaryCtrl {
+  /* @ngInject */
+  constructor($q, $translate, OvhHttp) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.OvhHttp = OvhHttp;
+  }
+
+  $onInit() {
+    const loadLanguage = this.$q
+      .when(this.me ? this.me.language : this.fetchLanguage())
+      .then((language) => {
+        this.language = language;
+      });
+    const loadBills = this.$q
+      .when(this.bills ? this.bills : this.fetchBills())
+      .then(({ data }) => {
+        this.bills = data;
+      });
+    const loadDebt = this.$q
+      .when(this.debt ? this.debt : this.fetchDebt())
+      .then(({ data }) => {
+        this.debt = data;
+      });
+
+    this.$translate.refresh().then(() => {
+      this.periods = [1, 3, 6].map((months) => ({
```

Comment:
Should `[1,3,6]` go into constants?

Reply:
since this code is not duplicated anywhere i'm not sure